### PR TITLE
feat: add to single line map arrowa to navigate between points

### PR DESCRIPTION
**Pull request by dorinzrihen**
# Description
https://github.com/hasadna/open-bus-map-search/issues/776
it's not solution with more zoom (i check with the docs and it look like it's the maximum)

## screenshots
image

Untitled video - Made with Clipchamp

### CODE REVIEW

File: src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    return Boolean(positions.at(i))
+  }
+
+  return (
+    <div className="map-footer-buttons">
+      <RightOutlined
+        className={`${checkIfValidStep(rightStep) ? '' : 'disabled'}`}
+        onClick={() => navigateMarkers(rightStep)}
+      />
+      <LeftOutlined
```
Comment (NoamGaash) [inline src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx:25]: we can add a "title" attribute for hover ("prev" / "next")

File: src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const leftStep = index - 1
+
+  const checkIfValidStep = (i: number) => {
+    return Boolean(positions.at(i))
+  }
+
+  return (
+    <div className="map-footer-buttons">
+      <RightOutlined
+        className={`${checkIfValidStep(rightStep) ? '' : 'disabled'}`}
```
Comment (NoamGaash) [inline src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx:22]: just an idea - we can hide the button if it's not relevant for this data point

**Review by NoamGaash — APPROVED** [review]
Perfect! Thank you

File: src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx (hunk 1 above)
Comment (dorinzrihen) [reply src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx:25]: I choose to use right/left-chevron instead cuz  'prev' 'next' depend on the direction 
image

Comment (NoamGaash) [reply src/pages/components/map-related/MapFooterButtons/MapFooterButtons.tsx:26]: the title attribute is visible to users, and should help them understand whether right will indicate the previous or the next item of the list.
In RTL website, it's common to see right arrows serving as "next" buttons, while on LTR website it's the opposite, that's why I feel having this title can be significant to the user experience
 
image

**Pull request merged**